src/features/corporate-training/api/useGetProgramOptions.ts

```diff
@@ -0,0 +1,165 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { useApiClient } from '~/providers/ApiClientProvider'
+import {
+  ProgramBranchResponse,
+  ProgramCategoryResponse,
+  ProgramLanguageResponse,
+  ProgramMethodResponse,
+  ProgramTopicResponse
+} from '../types/programOption'
+
+export const useGetProgramCategories = (
+  programCategoryParams: { title?: string },
+  options?: UseQueryOptions<
+    unknown,
+    unknown,
+    ProgramCategoryResponse[],
+    (string | undefined)[]
+  >
+) => {
+  const { title } = programCategoryParams
+
+  const { axiosStrapi, api } = useApiClient()
+
+  return useQuery(
+    ['program-categories', title],
```

Comment:
```suggestion
    ['strapi-program-categories', title],
```
buat data yang dari strapi kayanya harus di tambah keynya kaya gini, soalnya takutnya bentrok keynya sama yang dari api